acceptance/openstack/loadbalancer/v2/loadbalancers_test.go

```diff
@@ -46,10 +46,14 @@ func TestLoadbalancersCRUD(t *testing.T) {
 	th.AssertNoErr(t, err)
 	defer networking.DeleteSubnet(t, netClient, subnet.ID)
 
-	lb, err := CreateLoadBalancer(t, lbClient, subnet.ID)
+	lb, err := CreateLoadBalancerInSubnet(t, lbClient, subnet.ID)
 	th.AssertNoErr(t, err)
 	defer DeleteLoadBalancer(t, lbClient, lb.ID)
 
+	netlb, err := CreateLoadBalancerInNetwork(t, lbClient, network.ID)
```

Comment:
Would this have the test create a second loadbalancer? Since nothing further is happening to `netlb`, it won't do much good to have it created in parallel to the `lb` being created.

I would either create a whole new `TestLoadbalancersNetIDCRUD` test or just omit acceptance tests for this PR. It's always good to have acceptance tests, but this is just a single field being added `vip_network_id` and I'm fine with skipping the tests.